src/materialized/src/bin/materialized/main.rs

```diff
@@ -566,7 +566,13 @@ fn handle_panic(panic_info: &PanicInfo) {
         },
     };
 
-    log::error!(target: "panic", "{}: {}\n{:?}", thr_name, msg, Backtrace::new());
+    let location = if let Some(loc) = panic_info.location() {
+        format!("at {}:{}:{}", loc.file(), loc.line(), loc.column())
```

Comment:
`loc` implements `Display` so I think just `loc.to_string()` here and that will guarantee we match the stdlib: https://github.com/rust-lang/rust/blob/507bff92fadf1f25a830da5065a5a87113345163/library/std/src/panicking.rs#L203

Reply:
reasonable